data/json/recipes/recipe_ammo.json

```diff
@@ -2836,9 +2836,10 @@
     {"id":"CHEM","level":2,"amount":1}
   ], "tools": [
     [
-      [ "chemistry_set", 50 ],
       [ "hotplate", 50 ],
-      [ "toolset", 50 ]
+      [ "toolset", 50 ],
+      [ "fire", -1 ],
+      [ "char_smoker", 5 ]
```

Comment:
I changed all recipes in which you could drain charges from hotplate instead of chemistry set. Apart from this, I changed napalm and salt recipe because I think that you don't need to electrolyze anything to make them. Full list of changed recipes would be quite long.